gapminders/gapminders.ipynb

```diff
@@ -415,7 +436,7 @@
    "source": [
     "## Configure the template\n",
     "\n",
-    "Let us layout out the app in the nicely styled [FastListTemplate](https://panel.holoviz.org/reference/templates/FastListTemplate.html)."
+    "Let us layout out the app in the nicely styled with [FastListTemplate](https://panel.holoviz.org/reference/templates/FastListTemplate.html)."
```

Comment:
The grammar here doesn't sound right, I think.
```suggestion
    "Let us layout out the app in the nicely styled [FastListTemplate](https://panel.holoviz.org/reference/templates/FastListTemplate.html)."
```

Reply:
Thanks, I will make this change.

Reply:
I've made a PR with this change.